examples/jsm/objects/ReflectorForSSRPass.js

```diff
@@ -32,7 +32,7 @@ var Reflector = function ( geometry, options ) {
 	var textureHeight = options.textureHeight || 512;
 	var clipBias = options.clipBias || 0;
 	var shader = options.shader || Reflector.ReflectorShader;
-	var useDepthTexture = options.useDepthTexture
+	var useDepthTexture = options.useDepthTexture === true;
```

Comment:
I guess you want to rebase the commit so the changes from #21385 do not pop up in the diff.